CONTRIBUTING.md

```diff
@@ -35,10 +35,22 @@ If you choose to write some code, we have some conventions as follows.
 
 ## Guidelines
 
-### Coding Style
+### Checking the Format, Coding Style, and Type Hints
 
-Coding style is checked with [flake8](http://flake8.pycqa.org) and [isort](https://pycqa.github.io/isort/).
-Additional conventions are described in the [Wiki](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions).
+Code is formatted with [black](https://github.com/psf/black).
+Coding style is checked with [flake8](http://flake8.pycqa.org) and [isort](https://pycqa.github.io/isort/),
+and additional conventions are described in the [Wiki](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions).
+Type hints, [PEP484](https://www.python.org/dev/peps/pep-0484/), are checked with [mypy](http://mypy-lang.org/).
+
+You can check the format, coding style, and type hints, just by executing a script `checks.sh`.
+
+```bash
+# Without any fixes.
+sh checks.sh
```

Comment:
The shebang of `checks.sh` specifies `/bin/bash`, but `sh` is used here. If `checks.sh` does not require `/bin/bash`, could you change it to `/bin/sh` too?